http.go

```diff
@@ -1723,6 +1723,7 @@ func (resp *Response) brotliBody(level int) error {
 		resp.bodyRaw = nil
 	}
 	resp.Header.SetContentEncodingBytes(strBr)
+	resp.Header.Set("Vary", "Accept-Encoding")
```

Comment:
Sorry, but I'm thinking now it's probably better to use this. In case the user already set the Vary header before themselves.
```go
	if v := resp.Header.Peek(HeaderVary); !bytes.Contains(v, strAcceptEncoding) {
		resp.Header.SetBytesV(HeaderVary, append(v, (","+HeaderAcceptEncoding)...))
	} else {
		resp.Header.Add(HeaderVary, HeaderAcceptEncoding)
	}
```